install.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/sh -eu
+
+#
+# Installer.
+#
+
+INSTALL_ROOT=""
+MACHINE="nicole"
```

Comment:
If this is a build time default (when `-m` is not supplied), please add a comment explaining that.

Reply:
Added clarification to installer's help.